README.md

```diff
@@ -111,3 +111,51 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+If you have not yet run the setup environment script, run:
+```Shell
+./setup_env
+```
+Otherwise, if the command fails or you have run it before,
```

Comment:
I do agree. That said, I am getting a bit wary of doing so much in the setup env script, since I know that it can fail pretty easily if you have already run one or more steps of it? 
That said, there is nothing stopping a developer from going in and looking at the file.